fake_switches/cisco/command_processor/enabled.py

```diff
@@ -92,6 +92,39 @@ def do_show(self, *args):
                 self.write_line("Primary Secondary Type              Ports")
                 self.write_line("------- --------- ----------------- ------------------------------------------")
                 self.write_line("")
+        elif "etherchannel".startswith(args[0]) and len(args) == 2 and "summary".startswith(args[1]):
+            ports = sorted(self.switch_configuration.ports, key=lambda x: x.name)
+            port_channels = sorted(
+                [p for p in ports if isinstance(p, AggregatedPort)],
+                key=port_channel_number)
+            self.write_line("Flags:  D - down        P - bundled in port-channel")
+            self.write_line("        I - stand-alone s - suspended")
+            self.write_line("        H - Hot-standby (LACP only)")
+            self.write_line("        R - Layer3      S - Layer2")
+            self.write_line("        U - in use      f - failed to allocate aggregator")
+            self.write_line("")
+            self.write_line("        M - not in use, minimum links not met")
+            self.write_line("        u - unsuitable for bundling")
+            self.write_line("        w - waiting to be aggregated")
+            self.write_line("        d - default port")
+            self.write_line("")
+            self.write_line("")
+            self.write_line("Number of channel-groups in use: {}".format(len(port_channels)))
+            self.write_line("Number of aggregators:           {}".format(len(port_channels)))
+            self.write_line("")
+            self.write_line("Group  Port-channel  Protocol    Ports")
+            self.write_line("------+-------------+-----------+-----------------------------------------------")
+            for port_channel in port_channels:
+                po_number = port_channel_number(port_channel)
```

Comment:
making a variable of this is inconsistant with the other parts of the format at 121, maybe should be inline